GOVERNANCE.md

```diff
@@ -35,7 +35,7 @@ Peers:
 - Are expected to work on public branches of their forks and submit pull requests to the master branch.
 - Must submit pull requests for all their changes.
 - May label and close issues.
-- May only merge other people's pull requests that relate to compat data updates.
+- May only merge other people's pull requests that relate to compat data updates. (Note: Peers may merge changes to the files in the `browsers` directory — because those are compat data updates, not schema updates).
```

Comment:
suggestion: split this into two bullets, one about compat data and one about browser data

This also gives me an opportunity to make one fine distinction. Adding or removing browsers is _technically_ a schema change, but it seems to me that we might someday want to de-duplicate that.

```suggestion
- May merge other people's pull requests that relate to compat data updates.
- May merge other people's pull requests that relate to browser data updates (excluding the addition or removal of browsers).
```